susemanager-utils/testing/docker/master/uyuni-master-pgsql/postgresql.conf

```diff
@@ -9,7 +9,6 @@ lc_monetary = 'en_US.UTF-8'
 log_line_prefix = '%t %d %u '
 datestyle = 'iso, mdy'
 wal_keep_size = 1024
-wal_keep_segments = 64
```

Comment:
Is it because this doesn't exist at PostgreSQL13 anymore?

Reply:
yes, seems so

Reply:
was replaced by wal_keep_size https://www.postgresql.org/docs/13/release-13.html